trieste/acquisition/function/entropy.py

```diff
@@ -540,11 +540,14 @@ def __call__(self, x: TensorType) -> TensorType:
         L = tf.linalg.cholesky(
             B + noise_variance * tf.eye(len(self._pending_points), dtype=B.dtype)
         )  # need predictive variance of observations
-        A = tf.expand_dims(
-            self._model.covariance_between_points(  # type: ignore
-                tf.squeeze(x, -2), self._pending_points
+        A = tf.squeeze(
```

Comment:
was this a bug? did you perhaps create a test that catches it?

Reply:
this change is a consequence of me updating the covariance_between_points method. Now it always returns a tensor with an extra dimension (which would always be 1 unless it's a multi-output GPR)